tools/harness-thci/README.md

```diff
@@ -1,25 +1,26 @@
-OpenThread THCI
-===============
+THCI & Test Environment Setup
+=============================
 
-OpenThread THCI (Thread Host Controller Interface) is an implementation of the Python abstract class template "IThci",
+THCI (Thread Host Controller Interface) is an implementation of the Python abstract class template "IThci",
 which is used by Thread Test Harness Software to control OpenThread-based reference devices according to each test
 scenario.
 
-The current implementation is based on the CC2538 example platform and is included in the current Thread Test Harness
-Software release.
+Currently, there are two THCI implementations for OpenThread. One is for OpenThread CLI which is based on the CC2538 
+example platform and is included in the current Thread Test Harness Software release. The other is for OpenThread WpanCtl
+which is based on wpantund running on a linux host (For example: Raspberry Pi 3B) working with a NCP (Network Co-Processor) 
```

Comment:
nit: "For example:" -> "e.g."

Reply:
We should generally avoid using Latin abbreviations like i.e. and e.g., I would suggest:

(for example, Raspberry Pi 3B)

If we keep "for example", then we should do a search and update other uses of e.g. or i.e. in this README.